.github/workflows/build-test.yaml

```diff
@@ -206,36 +206,91 @@ jobs:
       - name: Run K3s and master in it
         run: tests/run-master-in-k3s.sh local/freeipa-server:${{ matrix.os }} freeipa-server-${{ matrix.os }}.tar
 
+  push-after-success-precondition:
+    name: Check pipeline settings
+    needs: [ test-docker, test-podman, test-rootless-podman, test-upgrade, test-k3s ]
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_TARGET_LIST: "${{ secrets.DOCKER_TARGET_LIST }}"
+      DOCKER_CREDENTIALS_FILE: "${{ secrets.DOCKER_CREDENTIALS_FILE }}"
+    steps:
+      - name: Check secrets
+        id: check
+        run: |
+          [ -e "${HOME}/.docker" ] || mkdir -p "${HOME}/.docker"
+          cat > "${HOME}/.docker/config.json" <<EOF
+          ${DOCKER_CREDENTIALS_FILE}
+          EOF
+          [ "${DOCKER_TARGET_LIST}" == "" ] && echo "Target list is empty" && echo "::set-output name=are-valid-credentials::false" && exit 0
+          IFS=":" read -a TARGET_LIST <<< "${DOCKER_TARGET_LIST}"
+          for base_name in "${TARGET_LIST[@]}"; do
+            docker login ${base_name%%/*} < /dev/null || {
+              echo "Can not log in to '${base_name%%/*}'' registry"
+              echo "::set-output name=are-valid-credentials::false"
+              exit 0
```

Comment:
I wouldn't do any `docker login`s here -- just check if the values are set.

Reply:
Thanks! Changed.